lib/tasks/fake_data.rake

```diff
@@ -72,7 +73,7 @@ namespace :fake_data do
       title: 'Any other information',
       comments: ['Former prison officer'] },
```

Comment:
Disabling it further it means that the ASSESSMENT_ANSWERS variable is also being ignored, I assume that's on purpose? If it wasn't ignored, I think the comments key would have to be wrapped in a hash? (If I've gotten the pattern right, that is)

Reply:
yep, one of the issues was that Rubocop wasn't happy about a constant being defined in a rake task - moving this disable statement forward circumvents that complaint.

(NB: this task is only used for generating fake data on dev systems)